aws/resource_aws_dynamodb_table.go

```diff
@@ -331,8 +331,11 @@ func resourceAwsDynamoDbTableCreate(d *schema.ResourceData, meta interface{}) er
 					time.Sleep(DYNAMODB_THROTTLE_SLEEP)
 					attemptCount += 1
 				case "LimitExceededException":
-					// If we're at resource capacity, error out without retry
-					if strings.Contains(awsErr.Message(), "Subscriber limit exceeded:") {
+					// If we're at resource capacity, error out without retry. e.g.
+					// Subscriber limit exceeded: There is a limit of 256 tables per subscriber
+					// Do not error out on this similar throttling message:
+					// Subscriber limit exceeded: Only 10 tables can be created, updated, or deleted simultaneously
+					if strings.Contains(awsErr.Message(), "Subscriber limit exceeded:") && !strings.Contains(awsErr.Message(), "simultaneously") {
```

Comment:
Do you mind tightening this a little bit, just to make sure we don't match an error we don't want to match? e.g. `can be created, updated, or deleted simultaneously`?

Reply:
Not a problem at all - will update shortly

Reply:
Not a problem at all - will update shortly